shuup/importer/templates/shuup/importer/admin/import.jinja

```diff
@@ -16,6 +16,11 @@
             <div class="col-md-6 col-md-offset-3">
                 {% if importer and importer.has_example_file() %}
                 <h2>{% trans %}Example files{% endtrans %}</h2>
+                <p>
+                    {% trans %}If there is a 'Ignore' column in the example file you are using.
+                    Make sure to leave the ignore column empty for all your rows,
+                    or else the row will not be imported.{% endtrans %}
```

Comment:
Make sure to leave the ignore column empty for all your rows, otherwise the rows won't be imported.